data-prepper-plugins/ruby-processor/.gitignore

```diff
@@ -0,0 +1,42 @@
+.gradle
```

Comment:
Do we need this file?

Reply:
Nope, sorry about that — will remove on next commit.